<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>551</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../paw 1.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="table_dons">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>441</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Ajouter un don</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>identifiant</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>type_don</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>quantite don</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>170</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>date d'accueil</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>20</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_type">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_quantite">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_d">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;
</string>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>150</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_date">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_e">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>113</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>identifiant donneur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 8pt &quot;Adobe Caslon Pro Bold&quot;;</string>
      </property>
      <property name="text">
       <string>identifiant employe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Afficher un don</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>80</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>chercher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>exporter PDF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>notification</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;
</string>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>160</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:white;
border-style:outset;
border-width:2px;
border-radius:10px;
border-color:white;</string>
      </property>
      <property name="text">
       <string>stat stockl</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Supprimer un don</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>20</y>
      <width>621</width>
      <height>381</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../Downloads/paw 1.jpg</pixmap>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>table_dons</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
